Src/VimCore/VimBuffer.fs

```diff
@@ -190,6 +190,7 @@ type internal VimBuffer
     let _keyMap = _vim.KeyMap
     let mutable _processingInputCount = 0
     let mutable _isClosed = false
+    let mutable _inSelectModeOneCommand = false

```

Comment:
Believe this state should be in `IVimTextBuffer` now. This would line up better with recent changes.